pkg/satellite/projectmembers.go

```diff
@@ -16,6 +16,8 @@ type ProjectMembers interface {
 	GetByMemberID(ctx context.Context, memberID uuid.UUID) (*ProjectMember, error)
 	// GetByProjectID is a method for querying project members from the database by projectID.
 	GetByProjectID(ctx context.Context, projectID uuid.UUID) ([]ProjectMember, error)
+	// GetPaged is a method for querying project members from the database by projectID, offset and limit.
+	GetPaged(ctx context.Context, projectID uuid.UUID, limit, offset int64) ([]ProjectMember, error)
```

Comment:
`GetByProjectIDPaged`, also is there a reason we have both `GetByProjectID` and the paged version?